WBooks/Rentals/RentalsViewModel.swift

```diff
@@ -9,4 +9,71 @@ import Foundation
 
 class RentalsViewModel {
     
+    let userId: Int = 10
+    var suggestions: [Suggestion]
+    var rents: [Book]
+    
+    init() {
+        suggestions = []
+        rents = []
+    }
+    
+    func getSuggestBooks(action: @escaping () -> () ) {
```

Comment:
In general if the closure don't return something, we can change -> () to -> Void

Reply:
Done